app/e2e/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "@app/e2e",
+    "version": "0.1.0",
+    "private": true,
+    "scripts": {
+        "test:e2e": "playwright test && yarn allure:generate-report",
+        "test:e2e:ci": "dotenv -v CI=1 -- yarn run test:e2e",
+        "test:e2e:headed": "playwright test --headed && yarn allure:generate-report",
+        "allure:generate-report": "allure generate ./allure-results --clean",
+        "allure:report": "allure open ./allure-report"
```

Comment:
do we need to scope these with `test:`?  I think it already makes sense that it's a test since we have to prefix with `yarn workspace @app/e2e`.

Maybe
* test
* ci:test
* debug (test headed because it's for debugging purposes)
* report (I'm not sure why generate-report and report are separate?)
Is there any reason we wouldn't want a clean report every time?

Also, could you update the documentation scripts.md?  It's pretty easy, it's just a markdown file

https://github.com/c2c-project/prytaneum/blob/dev/app/docs/docs/developing/scripts.md

Reply:
Yeah that makes sense, I'll update them and add to the documentation.

Reply:
Also for the report thing, one opens the report in the browser `allure:report` and the other is the one that actually generates the report. That's why it's two scripts.

Reply:
Could we potentially consolidate them both into view-report? Or are there times where we would need to generate them, but not view them?

Reply:
I mainly had them separated so we could generate and store them in the github actions, that way you can view the results of tests run on there by downloading the report. I could add another script called view report that just runs both if you think that would be better.